Controller/MediaAdminController.php

```diff
@@ -64,7 +65,7 @@ public function browserAction()
         $formView = $datagrid->getForm()->createView();
 
         // set the theme for the current Admin Form
-        $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
+        $this->setFormTheme($formView, $this->admin->getFilterTheme()[0]);
```

Comment:
Symfony 3.0 is php >=5.5.9 and Symfony >=3.4 is 7.0. It give nothing to keep the 5.3 dependency.